svm/tests/concurrent_tests.rs

```diff
@@ -152,6 +152,7 @@ fn svm_concurrent() {
     let mut check_data = vec![Vec::new(); THREADS];
     let read_account = Pubkey::new_unique();
     let mut account_data = AccountSharedData::default();
+    account_data.set_lamports(BALANCE);
     account_data.set_data(AMOUNT.to_le_bytes().to_vec());
```

Comment:
pointing this out for attention: i originally added the line to `load_account()` which returns `None` if the account is equal to `AccountSharedData::default()`, rather than merely having zero lamports, because of this test which sets up a zero-lamport read-only account. as long as we are sure no such accounts can actually exist, this change to the test is correct instead